app/models/mention_mailer.rb

```diff
@@ -6,10 +6,12 @@ class MentionMailer < ActionMailer::Base
   include ApplicationHelper
 
   def self.default_url_options
-    Mailer.default_url_options
+   h = Setting.host_name
+   h = h.to_s.gsub(%r{\/.*$}, '') unless Redmine::Utils.relative_url_root.blank?
```

Comment:
What does this regexp do? For me it was enough to just pass Setting.host_name to :host